backend/api/Areas/Admin/Models/AccessRequest/AccessRequestModel.cs

```diff
@@ -7,11 +6,9 @@ public class AccessRequestModel : Api.Models.BaseAppModel
     {
         #region Properties
         public long Id { get; set; }
-        public AccessRequestStatus Status { get; set; }
```

Comment:
Don't we still need the status? I think the note is gone from the schema, but the status is still there.

Reply:
While I updated many things I did not fully rewrite every model to match the new schema.